src/validation/func.rs

```diff
@@ -1623,19 +1402,16 @@ fn pop_value(
 	let actual_value = if stack_is_empty && is_stack_polymorphic {
 		StackValueType::Any
 	} else {
-		let value_stack_min = frame_stack
-			.top()
-			.expect("at least 1 topmost block")
-			.value_stack_len;
+		let value_stack_min = frame_stack.top().expect("at least 1 topmost block").value_stack_len;
```

Comment:
Again, better error message here please.

Reply:
Changed to "Expected a non-empty frame stack."